packages/preview/timeliney/0.0.1/README.md

```diff
@@ -0,0 +1,64 @@
+# Timeliney
+
+Create Gantt charts automatically with Typst!
+
+Here's a fully-featured example:
+
+```typst
+#import "@preview/timeliney:0.0.1"
+
+#timeliney.timeline(
+  show-grid: true,
+  {
+    import timeliney: *
+      
+    headerline(group(([*2023*], 4)), group(([*2024*], 4)))
+    headerline(
+      group(..range(4).map(n => strong("Q" + str(n + 1)))),
+      group(..range(4).map(n => strong("Q" + str(n + 1)))),
+    )
+  
+    taskgroup(title: [*Research*], {
+      task("Research the market", (0, 2), style: (stroke: 2pt + gray))
+      task("Conduct user surveys", (1, 3), style: (stroke: 2pt + gray))
+    })
+
+    taskgroup(title: [*Development*], {
+      task("Create mock-ups", (2, 3), style: (stroke: 2pt + gray))
+      task("Develop application", (3, 5), style: (stroke: 2pt + gray))
+      task("QA", (3.5, 6), style: (stroke: 2pt + gray))
+    })
+
+    taskgroup(title: [*Marketing*], {
+      task("Press demos", (3.5, 7), style: (stroke: 2pt + gray))
+      task("Social media advertising", (6, 7.5), style: (stroke: 2pt + gray))
+    })
+
+    milestone(
+      at: 3.75,
+      style: (stroke: (dash: "dashed")),
+      align(center, [
+        *Conference demo*\
+        Dec 2023
+      ])
+    )
+
+    milestone(
+      at: 6.5,
+      style: (stroke: (dash: "dashed")),
+      align(center, [
+        *App store launch*\
+        Aug 2024
+      ])
+    )
+  }
+)
+```
+
+![Example Gantt chart](sample.png)
```

Comment:
There's no image file

Reply:
I didn't include them because of this line in the submission guidelines:

> To keep the package small and fast to download, please exclude images for the README or PDF files with documentation from the bundle.

But I can change it to a link

Reply:
The wording could be clearer on this. You can include them in your commit, but you should add an `exclude` list to `typst.toml` so that they won't be part of the `tar.gz` archive that the compiler downloads. (Check out other packages to see how it works.) I will update the README.

Reply:
Ohh! I found it strange how `excludes` was worded but didn't associate it with the typst.toml file, I'll update this properly tomorrow

Reply:
I have another question, I'm using [mantys](https://github.com/jneug/typst-mantys) to generate the documentation, but since it's not available in the package repositories I ended up vendoring it into the repo. Should I include it here too?

Reply:
No, you can leave that out if you commit the pre-built manual.

Reply:
Done!